Sources/Router/Views/RouterLink.swift

```diff
@@ -14,13 +14,18 @@ public struct RouterLink<Label: View, Target: EnvironmentDependentRoute>: View {
     @usableFromInline
     var label: Label
     
+    @usableFromInline
+    var replaceRoot: Bool
+    
     /// Creates an instance that navigates to `destination`.
     /// - Parameters:
     ///   - destination: The navigation target route.
     ///   - label: A label describing the link.
+    ///   - replaceRoot: When set to `true`, the link will use the `replaceRoot` router method instead of `navigate`
     @inlinable
-    public init(to destination: Target, @ViewBuilder label: () -> Label) {
+    public init(to destination: Target, replaceRoot: Bool = false, @ViewBuilder label: () -> Label) {
```

Comment:
It all looks good, but my gut feeling says that adding a parameter in this place will potentially keep us expanding the signature for a while.

Reply:
Yes, I agree. When implementing this, I wasn't sure to be honest whether to add it as parameter like this, or as an additional environment value. Do you prefer the latter?

On the contrary, I'm not sure what other things one might configure on a router link. If the signature grows to be too large over time, in a future version we could of course still move the parameters to the environment.

Let me know what you prefer for now @Joannis, and I'll adjust it if needed.

Reply:
We can also make use of a `mutating func` that feels like a modifier.

Reply:
```swift
mutating func replaceRoot() -> Self {
  var copy = self
  copy.replaceRoot = true
  return copy
}
```

Reply:
I've just updated my comments, as the `mutating` part of the func would be annoying, not helpful.

Reply:
Alright, I'll implement it like that 🙂 Good idea 👌